lib/open_api_spex/plug/render_spec.ex

```diff
@@ -32,12 +32,21 @@ defmodule OpenApiSpex.Plug.RenderSpec do
   @impl Plug
   def init(opts), do: opts
 
-  @impl Plug
-  def call(conn, _opts) do
-    {spec, _} = PutApiSpec.get_spec_and_operation_lookup(conn)
-
-    conn
-    |> Plug.Conn.put_resp_content_type("application/json")
-    |> Plug.Conn.send_resp(200, @json_encoder.encode!(spec))
+  if @json_encoder do
```

Comment:
Is it possible to get this warning output at compile time rather than in the body of `call` ?

Reply:
We may call IO.warn in the else case?

Reply:
Changed to a compile-time warning.